### ISSUE TITLE
nbblack does not ignore jupyter magic cell lines

### ISSUE THREAD

**Issue by ericmjl** [vinayak-mehta/nbcommands#9]
@vinayak-mehta, first off, I want to say a big huge THANK YOU for this package! :tada: It'll be very helpful for me in interacting with my notebooks at the command line.

One thing I was very eager to try out was nbblack, because it has the ability to format every code cell in my notebook. The only problem that I encountered, though, was the code cell lines that have Jupyter magic inside of them. For example, if a code cell reads as:

```
%load_ext autoreload
%autoreload2
```

then nbblack will raise the following error: 

```
black.InvalidInput: Cannot parse 10:0: %load_ext autoreload
```

I was wondering if it might be possible to skip code magic when extracting code from each cell? A simple search to skip lines that contain `%` signs at the beginning might be the way to do it, I think, but having not dug further into the code base, I would defer to your knowledge on this.

**Comment by vinayak-mehta** [comment]
Thanks for raising an issue for this! I've fixed this in e8aa30b. Will create a bugfix release shortly :)

**Comment by vinayak-mehta** [comment]
Please install `0.3.1` with a `pip install --upgrade nbcommands` :)
